README.md

```diff
@@ -372,7 +372,7 @@ FOO_BAR=FOOBAR
 
 ### Quote character
 
-Detects if a value is wrapped in quotes:
+Detects if a value contains quote characters (`'` / `"`):
```

Comment:
Please also update the documentation [here](https://github.com/dotenv-linter/dotenv-linter/blob/master/docs/checks/quote_character.md).